src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,26 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['text'] = $column['default'] ?? '-';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['value'];
+    }
+@endphp
+
+<span>
```

Comment:
In Monsters I have two columns that will load this file: `color` and `color_picker`. But they show differently, one shows a preview and one shows the hex code... any idea why @munjaldevelopment ?

<img width="1230" alt="CleanShot 2022-10-18 at 14 09 22@2x" src="https://user-images.githubusercontent.com/1032474/196413863-1c446915-f484-4fcb-85a1-4ac806da7277.png">

Reply:
What I wanted out of this column is for it to show BOTH the preview (a circle with the color) AND the hex. Both at the same time. That covers the most ground:
- admin has a preview of the color (intuitive)
- admin also has the hex (so he can search, copy etc)

Reply:
Resolved color_picker & color as discussed in another thread.

Reply:
I... don't see this resolved @munjaldevelopment . Here's what I see:

<img width="1228" alt="CleanShot 2022-10-21 at 06 00 18@2x" src="https://user-images.githubusercontent.com/1032474/197108703-a7ef00d2-7dfd-4159-a386-d8593780295e.png">

But my expectations were to see:

<img width="1224" alt="CleanShot 2022-10-21 at 06 04 25@2x" src="https://user-images.githubusercontent.com/1032474/197109146-3b486027-9889-4865-8cce-4ec929033494.png">

Are we not on the same page here in terms of what it should look and work like? Do you want me to rephrase?